### PR TITLE
Add Nomspace integration

### PR DESCRIPTION
**Pull request by BriungRi**

### CODE REVIEW

**Comment by d-mooers** [comment]
Im looking into the build issue rn, but besides tht lgtm.  Pretty sure its just dependencies being finicky

File: src/components/Web3Status/index.tsx [hunk 1]
```diff
@@ -163,7 +173,7 @@ function Web3StatusInner() {
   } else {
     return (
       <Web3StatusConnect id="connect-wallet" onClick={() => connect().catch(console.warn)} faded={!address}>
-        <Text>{t('Connect to a wallet')}</Text>
+        <Text>{t('ConnectToAWallet')}</Text>
```
Comment (d-mooers) [inline src/components/Web3Status/index.tsx:176]: Could you add those spaces back between words?

File: src/components/Web3Status/index.tsx [hunk 2]
```diff
@@ -143,7 +150,10 @@ function Web3StatusInner() {
       <Web3StatusConnected id="web3-status-connected" onClick={toggleWalletModal} pending={hasPendingTransactions}>
         {hasPendingTransactions ? (
           <RowBetween>
-            <Text>{pending?.length} Pending</Text> <Loader stroke="white" />
+            <Text>
+              {pending?.length} {t('pending')}
+            </Text>{' '}
```
Comment (d-mooers) [inline src/components/Web3Status/index.tsx:155]: Nit: Could you remove the {' '} that was put there by autoformatter?

File: src/hooks/useAccountSummary.ts [hunk 3]
```diff
@@ -13,24 +14,38 @@ type AccountSummary = AsyncReturnType<AccountsWrapper['getAccountSummary']>
...
-      const accounts = await kit.contracts.getAccounts()
       try {
+        const accounts = await kit.contracts.getAccounts()
         const account = await accounts.signerToAccount(address)
         setSummary(await accounts.getAccountSummary(account))
       } catch (e) {
         console.error('Could not fetch account summary', e)
       }
+
+      const nom = new ENS({ provider, ensAddress: '0x3DE51c3960400A0F752d3492652Ae4A0b2A36FB3' })
```
Comment (d-mooers) [inline src/hooks/useAccountSummary.ts:40]: Nit: Could address be declared as ENS_ADDRESS in a constants file?

**Review by d-mooers — COMMENTED** [review]
some small nits but overall lgtm once the build passes

**Comment by BriungRi** [comment]
Screen Shot 2022-01-20 at 8 43 12 AM

**Pull request closed without merging**